DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +50,48 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def diskSpace(path, size = 'TB'):
+    """
+      Returns disk usage of the given path.
+      If no size is specified, terabytes will be used by default.
+    """
+
+    if size.upper() == "KB":
+      convert = 1024
+    elif size.upper() == "MB":
+      convert = 1.048576e6
+    elif size.upper() == "GB":
+      convert = 1.073741824e9
+    elif size.upper() == "TB":
+      convert = 1.099511627776e12
+    else:
+      return S_ERROR( "No valid size specified" )
+
+    st = os.statvfs(path)
```

Comment:
you need a try catch for all possible exception that might arrise. I am pretty sure there are OSError if the path does not exist for example